arkouda/pdarraycreation.py

```diff
@@ -744,6 +765,8 @@ def random_strings_lognormal(logmean : Union[float, int], logstd : Union[float,
         The number of strings to generate
     characters : (uppercase, lowercase, numeric, printable, binary)
         The set of characters to draw from
+    seed : int
+        Index for where to pull the first returned value
```

Comment:
"Value used to initialize random number generator"

Reply:
updated